packages/angular_devkit/build_angular/src/tools/sass/rebasing-importer.ts

```diff
@@ -74,6 +83,26 @@ abstract class UrlRebasingImporter implements Importer<'sync'> {
       updatedContents.update(start, end, rebasedUrl);
     }
 
+    for (const { start, end, value } of findImports(contents)) {
+      // Currently only provide directory information for known/common packages:
+      // * `@material/`
+      // * `@angular/`
+      // Comprehensive pre-resolution support may be added in the future.
+      if (!value.startsWith('@angular/') && !value.startsWith('@material/')) {
+        continue;
+      }
+
+      const resolvePath = stylesheetDirectory + ';' + value;
+
+      // Normalize path separators and escape characters
+      // https://developer.mozilla.org/en-US/docs/Web/CSS/url#syntax
+      const resolvedUrl =
+        'ng-package;' + resolvePath.replace(/\\/g, '/').replace(/[()\s'"]/g, '\\$&');
```

Comment:
Question: Would this break if the user named a file/project `ng-package`? I know it's unlikely, but not exactly implausible. Would something like `__ng-package__` be a little safer?